VogtObserver3/Test.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver3
+{
+    class Test
+    {
+        public static void Run()
+        {
+            Store store = new Store();
+            Store store2 = new Store();
+
+            Customer cust = new Customer("Julie", "Bed Bath & Beyond");
+            Customer cust2 = new Customer("Doug", "Hot Topic");
+
+            store.AddObserver(cust);
+            store.AddObserver(cust2);
```

Comment:
So they're observing ANY discounts in a given store it looks like? Not sure if that was intended, havent looked at the chegg.